src/components/AttachmentManager/AttachmentManager.test.js

```diff
@@ -153,12 +154,9 @@ describe('ImagePreview', () => {
         { attachmentType: 'image', url: 'https://nowhere/foo.png' }
       ],
       position: 1,
-      connectDragSource: i => i,
-      connectDragPreview: i => i,
-      connectDropTarget: i => i,
       removeImage: () => {}
     }
-    const wrapper = shallow(<ImagePreview.DecoratedComponent {...props} />)
+    const wrapper = render(<ImagePreview {...props} />)
```

Comment:
Good to see use of RTL, but would be nice to use some of it's more behavioral/"I am a user clicking through this" sorta stuff. We'll get there. I think there are some rough edges to still file off before it feels as easy as it should/could be.